test/e2e/kubectl.go

```diff
@@ -283,6 +284,49 @@ var _ = Describe("Kubectl client", func() {
 			})
 		})
 	})
+
+	Describe("Kubectl run", func() {
+		var nsFlag string
+		var rcName string
+
+		BeforeEach(func() {
+			nsFlag = fmt.Sprintf("--namespace=%v", ns)
+			rcName = "e2e-test-nginx-rc"
+		})
+
+		AfterEach(func() {
+			runKubectl("stop", "rc", rcName, nsFlag)
+		})
+
+		It("should create an rc from an image", func() {
+			image := "nginx"
+
+			By("running the image " + image)
+			runKubectl("run", rcName, "--image="+image, nsFlag)
+			By("verifying the rc " + rcName + " was created")
+			rc, err := c.ReplicationControllers(ns).Get(rcName)
```

Comment:
is it guaranteed that the rc should already exist by this point? or should there be a retry (maybe using `waitForReplicationController`?